src/number_theory/padics/padic_norm.lean

```diff
@@ -469,6 +469,12 @@ begin
     rwa nat.coprime_primes hp.1 hq.1, },
 end
 
+lemma valuation_prime_pow_eq_pow (p y : ℕ) [hp : fact p.prime] : padic_val_nat p (p ^ y) = y :=
```

Comment:
You don't use `hp` by name, so there is no need to name it. Also, this lemma name should include `padic_val_nat`:
```suggestion
lemma padic_val_nat_prime_pow (p y : ℕ) [fact p.prime] : padic_val_nat p (p ^ y) = y :=
```